### PR TITLE
Update search-suggestion-raw with 232 new terms

### PR DESCRIPTION
**Pull request by debjudy**
Update search-suggestion-raw with 232 new terms and sorted the list alphabetically

Please confirm the following steps are completed:

* [x] Choose the appropriate target branch:
  * Content
    * `preview` (approved content) <- *Content branch*
    * `main` (production) <- `preview`
* [x] Assign an appropriate reviewer:
  * *Content admin* or *Project lead* for merge to `preview` (approved content)
  * *Engineer* for merge to master (production)

:sunglasses: 

Updated file: https://github.com/18F/cv_faq/blob/search-suggestions-20201110/suggestions-raw.json

### CODE REVIEW

**Comment by debjudy** [comment]
Spurious htmlproofer failure

**Review by jonadecker — CHANGES_REQUESTED** [review]
Some dupes, some questions

**Comment by debjudy** [comment]
Spurious htmlproofer...again.

File: suggestions-raw.json [hunk 1]
```diff
@@ -1,1663 +1,1745 @@
...
+  "federal assistance",
+  "federal benefits",
+  "federal income tax",
+  "federal mandate",
+  "federal students loan",
+  "federal tax",
+  "federal tax filing",
+  "federal tax payment",
   "fema",
+  "fema assistance",
```
Comment (jonadecker) [inline suggestions-raw.json:579]: Adjacent dupes

File: suggestions-raw.json [hunk 2]
```diff
@@ -1,1663 +1,1745 @@
...
+  "fema assistance",
   "fever",
+  "fever temperature",
+  "fever and cough",
+  "fever temperature",
```
Comment (jonadecker) [inline suggestions-raw.json:584]: Split by one dupe

File: suggestions-raw.json [hunk 3]
```diff
@@ -1,1663 +1,1745 @@
...
-  "financial help",
   "financial",
+  "financial help",
+  "financial help for me and my family",
+  "financial aid",
+  "financial assistance",
+  "financial relief",
+  "firearms",
   "first case",
+  "first case",
```
Comment (jonadecker) [inline suggestions-raw.json:597]: Dupe

File: suggestions-raw.json [hunk 4]
```diff
@@ -1,1663 +1,1745 @@
...
   "fmla",
+  "food",
   "food assistance",
   "food delivery",
   "food for covid 19",
-  "food safety",
-  "food shopping",
-  "food stamps",
-  "food",
+  "food assistance",
```
Comment (jonadecker) [inline suggestions-raw.json:615]: dupe

**Comment by jonadecker** [comment]
Let's mob on this @debjudy and see what's going on

**Comment by debjudy** [comment]
@jonadecker hoping the 6th time is the charm.  Let me know if you see more dupes.

**Review by jonadecker — APPROVED** [review]
👍

**Pull request merged**